### DISCUSSION TITLE
Korath Food Aid

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9892] [Ideas]
One of the main reasons for the Korath Exiles to raid is to gain food. I think that a possible way to earn trust of the Korath Exiles would be to 'trade' food, either by dumping massive amounts of it in their systems or through some sort of in game mission questline (a possibility could be Human to Korath trade-food for technology).
Eventually, you could possibly land on their planets and buy their equipment-although you probably wouldn't be able to buy world ships .

**Comment by roadrunner56** [comment]
There's something like this in the works as part of a larger Korath Campaign: https://github.com/endless-sky/endless-sky/pull/8752
